### ISSUE TITLE
Installation failure on Ubuntu 18.04.3 LTS

### ISSUE THREAD

**Issue by greg-harhay** [ElucidataInc/ElMaven#1206]
I ran the apt-get commands to configure my environment without a problem:

sudo apt-get update
sudo apt-get install g++
sudo apt-get install libsqlite3-dev libboost-all-dev lcov libnetcdf-dev
sudo apt-get install qt5-qmake qtbase5-dev qtscript5-dev qtdeclarative5-dev libqt5multimedia5
sudo apt-get install libqt5multimedia5-plugins qtmultimedia5-dev libqt5webkit5-dev

I cloned the git repository and tried to make the executable but failed. Compliation failed at: 

**sentry.h:50:10: fatal error: uuid/uuid.h: No such file or directory**

I'm pasting the output from "make -j4" below. Suggestions to remedy?
 
$ qmake CONFIG+=release build.pro
$ make -j4

cd 3rdparty/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/3rdparty.pro CONFIG+=release ) && make -f Makefile 
make[1]: Entering directory '/home/greg/apps/ElMaven/3rdparty'
cd obiwarp/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/obiwarp/obiwarp.pro CONFIG+=release ) && make -f Makefile 
make[2]: Entering directory '/home/greg/apps/ElMaven/3rdparty/obiwarp'
compiling dynprog.cpp
compiling obiwarp.cpp
compiling mat.cpp
compiling vec.cpp
rm -f /home/greg/apps/build/libs/libobiwarp.a
ar cqs /home/greg/apps/build/libs/libobiwarp.a /home/greg/apps/build/tmp/obiwarp/obiwarp.o /home/greg/apps/build/tmp/obiwarp/dynprog.o /home/greg/apps/build/tmp/obiwarp/mat.o /home/greg/apps/build/tmp/obiwarp/vec.o
make[2]: Leaving directory '/home/greg/apps/ElMaven/3rdparty/obiwarp'
cd libneural/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/libneural/libneural.pro CONFIG+=release ) && make -f Makefile 
make[2]: Entering directory '/home/greg/apps/ElMaven/3rdparty/libneural'
compiling neuron.cc
compiling nnwork.cc
rm -f /home/greg/apps/build/libs/libneural.a
ar cqs /home/greg/apps/build/libs/libneural.a /home/greg/apps/build/tmp/neural/neuron.o /home/greg/apps/build/tmp/neural/nnwork.o
make[2]: Leaving directory '/home/greg/apps/ElMaven/3rdparty/libneural'
cd libcdfread/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/libcdfread/libcdfread.pro CONFIG+=release ) && make -f Makefile 
make[2]: Entering directory '/home/greg/apps/ElMaven/3rdparty/libcdfread'
compiling ms10aux.c
compiling ms10enum.c
compiling ms10io.c
rm -f /home/greg/apps/build/libs/libcdfread.a
ar cqs /home/greg/apps/build/libs/libcdfread.a /home/greg/apps/build/tmp/cdfread/ms10aux.o /home/greg/apps/build/tmp/cdfread/ms10enum.o /home/greg/apps/build/tmp/cdfread/ms10io.o
make[2]: Leaving directory '/home/greg/apps/ElMaven/3rdparty/libcdfread'
cd libcsvparser/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/libcsvparser/libcsvparser.pro CONFIG+=release ) && make -f Makefile 
make[2]: Entering directory '/home/greg/apps/ElMaven/3rdparty/libcsvparser'
compiling csvparser.c
rm -f /home/greg/apps/build/libs/libcsvparser.a
ar cqs /home/greg/apps/build/libs/libcsvparser.a /home/greg/apps/build/tmp/csv_parser/csvparser.o
make[2]: Leaving directory '/home/greg/apps/ElMaven/3rdparty/libcsvparser'
cd pugixml/src/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/pugixml/src/src.pro CONFIG+=release ) && make -f Makefile 
make[2]: Entering directory '/home/greg/apps/ElMaven/3rdparty/pugixml/src'
compiling pugixml.cpp
rm -f /home/greg/apps/build/libs/libpugixml.a
ar cqs /home/greg/apps/build/libs/libpugixml.a /home/greg/apps/build/tmp/pugixml/pugixml.o
make[2]: Leaving directory '/home/greg/apps/ElMaven/3rdparty/pugixml/src'
cd libpillow/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/libpillow/libpillow.pro CONFIG+=release ) && make -f Makefile 
make[2]: Entering directory '/home/greg/apps/ElMaven/3rdparty/libpillow'
compiling pch.h
compiling pch.h
compiling parser/parser.c
compiling parser/http_parser.c
parser.c: In function ‘thin_http_parser_execute’:
parser.rl:25:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:27:1: note: here
parser.rl:42:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:44:1: note: here
parser.rl:25:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:27:1: note: here
parser.rl:46:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:48:1: note: here
parser.rl:25:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:27:1: note: here
parser.c:256:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:258:1: note: here
parser.c:263:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:265:1: note: here
parser.c:270:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:272:1: note: here
parser.c:277:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:279:1: note: here
parser.c:284:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:286:1: note: here
parser.c:293:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:295:1: note: here
parser.c:300:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:302:1: note: here
parser.rl:38:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:40:1: note: here
parser.c:342:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:344:1: note: here
parser.c:370:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:372:1: note: here
parser.rl:28:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:30:1: note: here
parser.rl:33:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:35:1: note: here
parser.rl:33:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:35:1: note: here
parser.rl:46:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:48:1: note: here
parser.rl:25:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:27:1: note: here
parser.rl:25:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:27:1: note: here
parser.c:555:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:557:1: note: here
parser.c:568:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:570:1: note: here
parser.rl:25:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:27:1: note: here
parser.rl:25:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:27:1: note: here
parser.c:624:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:626:1: note: here
parser.c:639:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:641:1: note: here
parser.c:652:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:654:1: note: here
parser.rl:25:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:27:1: note: here
parser.c:684:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:686:1: note: here
parser.c:699:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:701:1: note: here
parser.c:712:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:714:1: note: here
parser.rl:66:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:68:1: note: here
parser.rl:52:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:54:1: note: here
parser.rl:52:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.rl:54:1: note: here
parser.c:784:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:786:1: note: here
parser.c:797:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:799:1: note: here
parser.c:810:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:812:1: note: here
parser.c:828:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:830:1: note: here
parser.c:846:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:848:1: note: here
parser.c:864:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:866:1: note: here
parser.c:882:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:884:1: note: here
parser.c:900:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:902:1: note: here
parser.c:918:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:920:1: note: here
parser.c:936:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:938:1: note: here
parser.c:954:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:956:1: note: here
parser.c:972:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:974:1: note: here
parser.c:990:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:992:1: note: here
parser.c:1008:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:1010:1: note: here
parser.c:1026:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:1028:1: note: here
parser.c:1044:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:1046:1: note: here
parser.c:1062:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:1064:1: note: here
parser.c:1080:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:1082:1: note: here
parser.c:1098:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:1100:1: note: here
parser.c:1116:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:1118:1: note: here
parser.c:1134:5: warning: this statement may fall through [-Wimplicit-fallthrough=]
parser.c:1136:1: note: here
compiling HttpServer.cpp
compiling HttpHandler.cpp
compiling HttpHandlerQtScript.cpp
compiling HttpHelpers.cpp
compiling HttpsServer.cpp
compiling HttpHandlerSimpleRouter.cpp
compiling HttpConnection.cpp
compiling HttpHandlerProxy.cpp
generating /home/greg/apps/ElMaven/../build/tmp/pillow//moc_predefs.h
compiling HttpHeader.cpp
moc HttpServer.h
moc HttpHandler.h
moc HttpHandlerQtScript.h
moc HttpsServer.h
moc HttpHandlerSimpleRouter.h
moc HttpConnection.h
moc HttpHandlerProxy.h
moc HttpClient.h
moc HttpClient.cpp
compiling /home/greg/apps/build/tmp/pillow/moc_HttpServer.cpp
compiling /home/greg/apps/build/tmp/pillow/moc_HttpHandler.cpp
compiling /home/greg/apps/build/tmp/pillow/moc_HttpHandlerQtScript.cpp
compiling /home/greg/apps/build/tmp/pillow/moc_HttpsServer.cpp
compiling /home/greg/apps/build/tmp/pillow/moc_HttpHandlerSimpleRouter.cpp
compiling /home/greg/apps/build/tmp/pillow/moc_HttpConnection.cpp
compiling /home/greg/apps/build/tmp/pillow/moc_HttpHandlerProxy.cpp
compiling /home/greg/apps/build/tmp/pillow/moc_HttpClient.cpp
compiling HttpClient.cpp
rm -f /home/greg/apps/build/libs/libpillowcore.a
ar cqs /home/greg/apps/build/libs/libpillowcore.a /home/greg/apps/build/tmp/pillow/parser.o /home/greg/apps/build/tmp/pillow/http_parser.o /home/greg/apps/build/tmp/pillow/HttpServer.o /home/greg/apps/build/tmp/pillow/HttpHandler.o /home/greg/apps/build/tmp/pillow/HttpHandlerQtScript.o /home/greg/apps/build/tmp/pillow/HttpHelpers.o /home/greg/apps/build/tmp/pillow/HttpsServer.o /home/greg/apps/build/tmp/pillow/HttpHandlerSimpleRouter.o /home/greg/apps/build/tmp/pillow/HttpConnection.o /home/greg/apps/build/tmp/pillow/HttpHandlerProxy.o /home/greg/apps/build/tmp/pillow/HttpClient.o /home/greg/apps/build/tmp/pillow/HttpHeader.o /home/greg/apps/build/tmp/pillow/moc_HttpServer.o /home/greg/apps/build/tmp/pillow/moc_HttpHandler.o /home/greg/apps/build/tmp/pillow/moc_HttpHandlerQtScript.o /home/greg/apps/build/tmp/pillow/moc_HttpsServer.o /home/greg/apps/build/tmp/pillow/moc_HttpHandlerSimpleRouter.o /home/greg/apps/build/tmp/pillow/moc_HttpConnection.o /home/greg/apps/build/tmp/pillow/moc_HttpHandlerProxy.o /home/greg/apps/build/tmp/pillow/moc_HttpClient.o
make[2]: Leaving directory '/home/greg/apps/ElMaven/3rdparty/libpillow'
cd libpls/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/libpls/libpls.pro CONFIG+=release ) && make -f Makefile 
make[2]: Entering directory '/home/greg/apps/ElMaven/3rdparty/libpls'
compiling pls.cpp
compiling utility.cpp
rm -f /home/greg/apps/build/libs/libpls.a
ar cqs /home/greg/apps/build/libs/libpls.a /home/greg/apps/build/tmp/pls/pls.o /home/greg/apps/build/tmp/pls/utility.o
make[2]: Leaving directory '/home/greg/apps/ElMaven/3rdparty/libpls'
cd Logger/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/Logger/Logger.pro CONFIG+=release ) && make -f Makefile 
make[2]: Entering directory '/home/greg/apps/ElMaven/3rdparty/Logger'
g++ -c -pipe -std=c++11 -O2 -fPIC -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -I. -Ispdlog -I/home/greg/anaconda3/include/qt -I/home/greg/anaconda3/include/qt/QtGui -I/home/greg/anaconda3/include/qt/QtCore -I/home/greg/apps/build/tmp/logger -I/home/greg/anaconda3/mkspecs/linux-g++ -o /home/greg/apps/build/tmp/logger/elmavenlogger.o elmavenlogger.cpp
rm -f /home/greg/apps/build/libs/libLogger.a
ar cqs /home/greg/apps/build/libs/libLogger.a /home/greg/apps/build/tmp/logger/elmavenlogger.o
make[2]: Leaving directory '/home/greg/apps/ElMaven/3rdparty/Logger'
cd ErrorHandling/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/ErrorHandling/ErrorHandling.pro CONFIG+=release ) && make -f Makefile 
make[2]: Entering directory '/home/greg/apps/ElMaven/3rdparty/ErrorHandling'
g++ -c -pipe -std=c++11 -O2 -fPIC -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -I. -I../Logger -I/home/greg/anaconda3/include/qt -I/home/greg/anaconda3/include/qt/QtGui -I/home/greg/anaconda3/include/qt/QtCore -I/home/greg/apps/build/tmp/error_handling -I/home/greg/anaconda3/mkspecs/linux-g++ -o /home/greg/apps/build/tmp/error_handling/MavenException.o MavenException.cpp
g++ -c -pipe -std=c++11 -O2 -fPIC -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -I. -I../Logger -I/home/greg/anaconda3/include/qt -I/home/greg/anaconda3/include/qt/QtGui -I/home/greg/anaconda3/include/qt/QtCore -I/home/greg/apps/build/tmp/error_handling -I/home/greg/anaconda3/mkspecs/linux-g++ -o /home/greg/apps/build/tmp/error_handling/errorcodes.o errorcodes.cpp
rm -f /home/greg/apps/build/libs/libErrorHandling.a
ar cqs /home/greg/apps/build/libs/libErrorHandling.a /home/greg/apps/build/tmp/error_handling/MavenException.o /home/greg/apps/build/tmp/error_handling/errorcodes.o
make[2]: Leaving directory '/home/greg/apps/ElMaven/3rdparty/ErrorHandling'
cd libsvm/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/libsvm/libsvm.pro CONFIG+=release ) && make -f Makefile 
make[2]: Entering directory '/home/greg/apps/ElMaven/3rdparty/libsvm'
g++ -c -pipe -O2 -fPIC -Wall -W -D_REENTRANT -DQT_NO_DEBUG -DQT_GUI_LIB -DQT_CORE_LIB -I. -I. -I/home/greg/anaconda3/include/qt -I/home/greg/anaconda3/include/qt/QtGui -I/home/greg/anaconda3/include/qt/QtCore -I/home/greg/apps/build/tmp/svm -I/home/greg/anaconda3/mkspecs/linux-g++ -o /home/greg/apps/build/tmp/svm/svm.o svm.cpp
rm -f /home/greg/apps/build/libs/libsvm.a
ar cqs /home/greg/apps/build/libs/libsvm.a /home/greg/apps/build/tmp/svm/svm.o
make[2]: Leaving directory '/home/greg/apps/ElMaven/3rdparty/libsvm'
cd NimbleDSP/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/NimbleDSP/NimbleDSP.pro CONFIG+=release ) && make -f Makefile 
make[2]: Entering directory '/home/greg/apps/ElMaven/3rdparty/NimbleDSP'
cd src/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/NimbleDSP/src/src.pro CONFIG+=release ) && make -f Makefile 
make[3]: Entering directory '/home/greg/apps/ElMaven/3rdparty/NimbleDSP/src'
gcc -c -pipe -O2 -fPIC -w  -I. -I../../../src -isystem /usr/local/include -I/home/greg/anaconda3/mkspecs/linux-g++ -o /home/greg/apps/build/tmp/nimble/kiss_fft.o kiss_fft.c
gcc -c -pipe -O2 -fPIC -w  -I. -I../../../src -isystem /usr/local/include -I/home/greg/anaconda3/mkspecs/linux-g++ -o /home/greg/apps/build/tmp/nimble/kiss_fftr.o kiss_fftr.c
rm -f /home/greg/apps/build/libs/libNimbleDSP.a
ar cqs /home/greg/apps/build/libs/libNimbleDSP.a /home/greg/apps/build/tmp/nimble/kiss_fft.o /home/greg/apps/build/tmp/nimble/kiss_fftr.o
make[3]: Leaving directory '/home/greg/apps/ElMaven/3rdparty/NimbleDSP/src'
make[2]: Leaving directory '/home/greg/apps/ElMaven/3rdparty/NimbleDSP'
cd libmgf/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/libmgf/libmgf.pro CONFIG+=release ) && make -f Makefile 
make[2]: Entering directory '/home/greg/apps/ElMaven/3rdparty/libmgf'
cd src/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/3rdparty/libmgf/src/src.pro CONFIG+=release ) && make -f Makefile 
make[3]: Entering directory '/home/greg/apps/ElMaven/3rdparty/libmgf/src'
bash parsergen.sh
g++ -c -pipe -std=c++11 -O2 -fPIC -std=gnu++11 -w  -I. -I. -I../mgf -isystem /usr/local/include -I/home/greg/anaconda3/mkspecs/linux-g++ -o /home/greg/apps/build/tmp/libmgf/MgfFile.o MgfFile.cpp
g++ -c -pipe -std=c++11 -O2 -fPIC -std=gnu++11 -w  -I. -I. -I../mgf -isystem /usr/local/include -I/home/greg/anaconda3/mkspecs/linux-g++ -o /home/greg/apps/build/tmp/libmgf/MgfHeader.o MgfHeader.cpp
g++ -c -pipe -std=c++11 -O2 -fPIC -std=gnu++11 -w  -I. -I. -I../mgf -isystem /usr/local/include -I/home/greg/anaconda3/mkspecs/linux-g++ -o /home/greg/apps/build/tmp/libmgf/MgfSpectrum.o MgfSpectrum.cpp
The scanner and parser are up to date.
g++ -c -pipe -std=c++11 -O2 -fPIC -std=gnu++11 -w  -I. -I. -I../mgf -isystem /usr/local/include -I/home/greg/anaconda3/mkspecs/linux-g++ -o /home/greg/apps/build/tmp/libmgf/Driver.o Driver.cpp
g++ -c -pipe -std=c++11 -O2 -fPIC -std=gnu++11 -w  -I. -I. -I../mgf -isystem /usr/local/include -I/home/greg/anaconda3/mkspecs/linux-g++ -o /home/greg/apps/build/tmp/libmgf/Parser.o Parser.cpp
g++ -c -pipe -std=c++11 -O2 -fPIC -std=gnu++11 -w  -I. -I. -I../mgf -isystem /usr/local/include -I/home/greg/anaconda3/mkspecs/linux-g++ -o /home/greg/apps/build/tmp/libmgf/Scanner.o Scanner.cpp
rm -f /home/greg/apps/build/libs/libmgf.a
ar cqs /home/greg/apps/build/libs/libmgf.a /home/greg/apps/build/tmp/libmgf/Driver.o /home/greg/apps/build/tmp/libmgf/MgfFile.o /home/greg/apps/build/tmp/libmgf/MgfHeader.o /home/greg/apps/build/tmp/libmgf/MgfSpectrum.o /home/greg/apps/build/tmp/libmgf/Parser.o /home/greg/apps/build/tmp/libmgf/Scanner.o
make[3]: Leaving directory '/home/greg/apps/ElMaven/3rdparty/libmgf/src'
make[2]: Leaving directory '/home/greg/apps/ElMaven/3rdparty/libmgf'
make[1]: Leaving directory '/home/greg/apps/ElMaven/3rdparty'
cd src/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/src/src.pro CONFIG+=release ) && make -f Makefile 
make[1]: Entering directory '/home/greg/apps/ElMaven/src'
cd core/libmaven/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/src/core/libmaven/libmaven.pro CONFIG+=release ) && make -f Makefile 
make[2]: Entering directory '/home/greg/apps/ElMaven/src/core/libmaven'
compiling base64.cpp
compiling mzMassCalculator.cpp
compiling mzPatterns.cpp
compiling mzSample.cpp
compiling mzUtils.cpp
compiling statistics.cpp
compiling elementMass.cpp
compiling mzFit.cpp
compiling mzAligner.cpp
compiling mzMassSlicer.cpp
compiling PeakGroup.cpp
compiling Fragment.cpp
compiling EIC.cpp
compiling Scan.cpp
compiling SRMList.cpp
compiling Peak.cpp
compiling Compound.cpp
compiling savgol.cpp
compiling SavGolSmoother.cpp
compiling PeakDetector.cpp
compiling mavenparameters.cpp
compiling classifier.cpp
compiling classifierNaiveBayes.cpp
compiling classifierNeuralNet.cpp
compiling csvreports.cpp
compiling comparesampleslogic.cpp
compiling isotopelogic.cpp
compiling eiclogic.cpp
compiling databases.cpp
compiling Peptide.cpp
compiling PolyAligner.cpp
compiling jsonReports.cpp
compiling masscutofftype.cpp
compiling peakFiltering.cpp
compiling groupFiltering.cpp
compiling isotopeDetection.cpp
compiling datastructures/mzSlice.cpp
compiling groupClassifier.cpp
compiling groupFeatures.cpp
compiling svmPredictor.cpp
compiling zlib.cpp
rm -f /home/greg/apps/build/libs/libmaven.a
ar cqs /home/greg/apps/build/libs/libmaven.a /home/greg/apps/build/tmp/maven/base64.o /home/greg/apps/build/tmp/maven/mzMassCalculator.o /home/greg/apps/build/tmp/maven/mzPatterns.o /home/greg/apps/build/tmp/maven/mzSample.o /home/greg/apps/build/tmp/maven/mzUtils.o /home/greg/apps/build/tmp/maven/statistics.o /home/greg/apps/build/tmp/maven/elementMass.o /home/greg/apps/build/tmp/maven/mzFit.o /home/greg/apps/build/tmp/maven/mzAligner.o /home/greg/apps/build/tmp/maven/mzMassSlicer.o /home/greg/apps/build/tmp/maven/PeakGroup.o /home/greg/apps/build/tmp/maven/Fragment.o /home/greg/apps/build/tmp/maven/EIC.o /home/greg/apps/build/tmp/maven/Scan.o /home/greg/apps/build/tmp/maven/SRMList.o /home/greg/apps/build/tmp/maven/Peak.o /home/greg/apps/build/tmp/maven/Compound.o /home/greg/apps/build/tmp/maven/savgol.o /home/greg/apps/build/tmp/maven/SavGolSmoother.o /home/greg/apps/build/tmp/maven/PeakDetector.o /home/greg/apps/build/tmp/maven/mavenparameters.o /home/greg/apps/build/tmp/maven/classifier.o /home/greg/apps/build/tmp/maven/classifierNaiveBayes.o /home/greg/apps/build/tmp/maven/classifierNeuralNet.o /home/greg/apps/build/tmp/maven/csvreports.o /home/greg/apps/build/tmp/maven/comparesampleslogic.o /home/greg/apps/build/tmp/maven/isotopelogic.o /home/greg/apps/build/tmp/maven/eiclogic.o /home/greg/apps/build/tmp/maven/databases.o /home/greg/apps/build/tmp/maven/Peptide.o /home/greg/apps/build/tmp/maven/PolyAligner.o /home/greg/apps/build/tmp/maven/jsonReports.o /home/greg/apps/build/tmp/maven/masscutofftype.o /home/greg/apps/build/tmp/maven/peakFiltering.o /home/greg/apps/build/tmp/maven/groupFiltering.o /home/greg/apps/build/tmp/maven/isotopeDetection.o /home/greg/apps/build/tmp/maven/mzSlice.o /home/greg/apps/build/tmp/maven/groupClassifier.o /home/greg/apps/build/tmp/maven/groupFeatures.o /home/greg/apps/build/tmp/maven/svmPredictor.o /home/greg/apps/build/tmp/maven/zlib.o
make[2]: Leaving directory '/home/greg/apps/ElMaven/src/core/libmaven'
cd common/ && ( test -e Makefile || /home/greg/anaconda3/bin/qmake -o Makefile /home/greg/apps/ElMaven/src/common/common.pro CONFIG+=release ) && make -f Makefile 
make[2]: Entering directory '/home/greg/apps/ElMaven/src/common'
compiling downloadmanager.cpp
compiling logger.cpp
compiling analytics.cpp
compiling mixpanel.cpp
In file included from mixpanel.cpp:11:0:
sentry.h:50:10: fatal error: uuid/uuid.h: No such file or directory
 #include <uuid/uuid.h>
          ^~~~~~~~~~~~~
compilation terminated.
Makefile:1155: recipe for target '/home/greg/apps/build/tmp/common/mixpanel.o' failed
make[2]: *** [/home/greg/apps/build/tmp/common/mixpanel.o] Error 1
make[2]: *** Waiting for unfinished jobs....
analytics.cpp: In member function ‘void Analytics::httpPost(QUrlQuery)’:
analytics.cpp:108:20: warning: unused variable ‘reply’ [-Wunused-variable]
     QNetworkReply *reply = manager->post(req, data);
                    ^~~~~
In file included from downloadmanager.cpp:6:0:
downloadmanager.h: In constructor ‘DownloadManager::DownloadManager()’:
downloadmanager.h:41:20: warning: ‘DownloadManager::_reply’ will be initialized after [-Wreorder]
     QNetworkReply* _reply;
                    ^~~~~~
downloadmanager.h:19:10: warning:   ‘bool DownloadManager::err’ [-Wreorder]
     bool err;
          ^~~
downloadmanager.cpp:10:1: warning:   when initialized here [-Wreorder]
 DownloadManager::DownloadManager():
 ^~~~~~~~~~~~~~~
make[2]: Leaving directory '/home/greg/apps/ElMaven/src/common'
Makefile:92: recipe for target 'sub-common-make_first-ordered' failed
make[1]: *** [sub-common-make_first-ordered] Error 2
make[1]: Leaving directory '/home/greg/apps/ElMaven/src'
Makefile:89: recipe for target 'sub-src-make_first-ordered' failed
make: *** [sub-src-make_first-ordered] Error 2

**Comment by saifulbkhan** [comment]
Hi @greg-harhay,

Sorry you had to face this issue. This made it to our `develop` branch some time ago. I will make fix this soon, but until then can you please try wrapping the `#include "sentry.h"` code in file `src/common/mixpanel.cpp` at [line 11](https://github.com/ElucidataInc/ElMaven/blob/develop/src/common/mixpanel.cpp#L11) as:
```C++
#ifdef __OSX_AVAILABLE
#ifndef DEBUG
#include "sentry.h"
#endif
#endif
```
and see if compilation proceeds for you?

**Comment by greg-harhay** [comment]
Wrapping #include "sentry.h" with the code in src/common/mixpanel.cpp solved the problem in El-MAVEN v0.9.1-23. Up and running now. Thank you
